newIDE/app/src/UI/EmptyPlaceholder.js

```diff
@@ -52,7 +53,15 @@ export const EmptyPlaceholder = (props: Props) => (
             primary
             onClick={props.onAdd}
             disabled={!!props.isLoading}
-            icon={props.isLoading ? <CircularProgress size={24} /> : <Add />}
+            icon={
```

Comment:
Can we also refactor the name of the action to be `onAction` rather than `onAdd`?
Then the action is either "add" by default, or something else if another icon is provided

Reply:
weird, maybe just because VSCode is lost, it's only used 2 or 3 times across the app though, so the refactor could be done manually